### DISCUSSION TITLE
Request adding ZMK support for Waterfowl

### DISCUSSION THREAD

**Discussion by yooneo** [manna-harbour/miryoku#210] [Q&A] [answered]
Please consider adding ZMK miryoku support for the Waterfowl keyboard.

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the request! It was on [the list](https://github.com/manna-harbour/miryoku/discussions/81), but it's added now in https://github.com/manna-harbour/miryoku_zmk/commit/fd0602354860221c94e0cfcf71e4ec54096895df. Here's a sample build run: https://github.com/manna-harbour/miryoku_zmk/actions/runs/4972512880.

(@JW2586)

**Reply by yooneo** [reply]
Thank you! I just tested it on mine, and it works great!
